test/checks/forms/fieldset.js

```diff
@@ -43,8 +43,8 @@ describe('fieldset', function () {
 		});
 
 		it('should return false if the group has no legend element', function () {
-			fixture.innerHTML = '<fieldset><input type="' + type + '" id="target" name="uniqueyname">' +
-				'<input type="' + type + '" name="uniqueyname"></fieldset>';
+			fixtureSetup('<fieldset><input type="' + type + '" id="target" name="uniqueyname">' +
```

Comment:
this test here should work when the fieldset has been found by going up the slotted content - see the example below